environment-dev.yml

```diff
@@ -11,9 +11,9 @@ dependencies:
   - nlohmann_json=3.11.2
   - cppzmq
   - xtl
-  - clangdev >=16,<17
+  - CppInterOp
   - pugixml
-  - cpp-argparse
+  - cpp-argparse <3.0
```

Comment:
I wouldn't made this change. If you allow this to include version 3.0 then you can cover arm builds, as I updated the conda feedstock of cpp-argparse to include a Linux Arm build here https://github.com/conda-forge/cpp-argparse-feedstock/pull/5 , and am currently waiting for my PR in the xeus-cpp conda feedstock to be approved, so that xeus-cpp has arm recipes available here https://github.com/conda-forge/xeus-cpp-feedstock/pull/10

Reply:
Can you do that change in a separate PR? I pinged the folks about the pending PR on xeus-cpp.

Reply:
Yes. I will wait for the xeus-cpp PR to be merged, and then put this change in as a separate PR.

Reply:
Ok.

Reply:
Would be great if you could just reflect these versioning changes and the clangdev change in the `README.md` file too. It displays the following currently 
`mamba install notebook cmake cxx-compiler xeus-zmq nlohmann_json=3.11.2 cppzmq xtl jupyterlab clangdev=16 cpp-argparse pugixml doctest -c conda-forge`

Reply:
The feedstock PRs will be merged momentarily. I think that it is fine now to put `argparse <3.1` (so that we can take patch releases).

Reply:
```suggestion
  - cpp-argparse <3.1
```